libethcore/Transaction.cpp

```diff
@@ -64,10 +64,18 @@ TransactionBase::TransactionBase(bytesConstRef _rlpData, CheckTransaction _check
 			BOOST_THROW_EXCEPTION(InvalidTransactionFormat() << errinfo_comment("transaction data RLP must be an array"));
 
 		m_data = rlp[field = 5].toBytes();
-		byte v = rlp[field = 6].toInt<byte>() - 27;
+
+		byte v = rlp[field = 6].toInt<byte>();
```

Comment:
it looks like field is used to make it more readable 
rlp[field = 1]  istead of rlp[1]